DC-SLES-container-guide

```diff
@@ -0,0 +1,16 @@
+## ---------------------------- 
```

Comment:
We will have the DC file name as part of our URLs, so I'd suggest making it consistent with other DC file names and rename to `DC-SLES-container`.